frontend/src/environments/environment.development.ts

```diff
@@ -4,8 +4,8 @@
 
 export const environment = {
```

Comment:
please leave it as it was if it doesnt work for you than change it only locally on your code